src/services/flow-service.js

```diff
@@ -95,7 +99,7 @@ const isFlowExist = async function (flowName, transaction) {
   }, transaction);
 
   if (flow) {
-    throw new Errors.ValidationError("Bad Request: Flow with the same name already exists")
+    throw new Errors.ValidationError("Bad Request: Flow with the name" + flowName + "already exists")
```

Comment:
I think we might need to create separate file with all validation errors so they look similar to each other

Reply:
good idea

Reply:
who will do that?